interfaces/cython/cantera/examples/reactors/ic_engine.py

```diff
@@ -106,7 +111,8 @@ def piston_speed(t):
 inlet_delta = np.mod(inlet_close - inlet_open, 4 * np.pi)
 inlet_valve.valve_coeff = inlet_valve_coeff
 inlet_valve.set_time_function(
-    lambda t: np.mod(crank_angle(t) - inlet_open, 4 * np.pi) < inlet_delta)
+    lambda t: np.mod(crank_angle(t) - inlet_open, 4 * np.pi) < inlet_delta
+)
```

Comment:
I don't think the original version was PEP8 compliant. My interpretation of it is that the closing paren can be on the same line as a multi-line construct when the text is aligned with the first line:
```
foo = func(var1, var2,
           var3, var4)
```
but if the construct starts on the next line, the closing paren should be in its own line:
```
foo = func(
    var1, var2,
    var3, var4,
)
Or
foo = func(
    var1, var2,
    var3, var4,
    )
```
There's nothing explicit about this in the PEP, but there aren't any examples in the whitespace section of the PEP that use the format that was originally on this line.

Aside from appealing to the PEP, I personally find the new version much easier to read, because the closing paren is much easier to find. The original way I find it easy to read past that paren and interpret some following code as still belonging to the function.

Reply:
I reverted this change.